build.sbt

```diff
@@ -41,21 +43,6 @@ lazy val root = Project("spark-redshift", file("."))
     name := "spark-redshift",
     organization := "io.github.spark-redshift-community",
     scalaVersion := "2.12.11",
-    sparkVersion := "3.0.1",
-    testSparkVersion := sys.props.get("spark.testVersion").getOrElse(sparkVersion.value),
-
-    // Spark 2.4.x should be compatible with hadoop >= 2.7.x
-    // https://spark.apache.org/downloads.html
-    testHadoopVersion := sys.props.get("hadoop.testVersion").getOrElse("3.2.1"),
-
-    // DON't UPGRADE AWS-SDK-JAVA if not compatible with hadoop version
-    // https://stackoverflow.com/a/49510602/2544874
-    // https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-aws/2.7.7
-    testAWSJavaSDKVersion := sys.props.get("aws.testVersion").getOrElse("1.11.375"),
-
-    spName := "spark-redshift-community/spark-redshift",
```

Comment:
Why are we removing this?

Reply:
the `spName`?  That was a property that the sbt-spark-package plugin injected into the build.  Details here https://github.com/databricks/sbt-spark-package/blob/master/src/main/scala/sbtsparkpackage/SparkPackagePlugin.scala

Since we're removing sbt-spark-package then we also need to remove all the "sp" prefix things.  I don't think this spName was doing anything important behind the scenes, but take a look at the link above and double check.